lib/assesments/assessment_2/fi_as2_saleh/screens/profile/widgets/fi_as2_saleh_stats.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class StatsColumn extends StatelessWidget {
+  final String name;
+  final String number;
```

Comment:
instead of number you can use something like followers etc, this will describe it better